pytest is failing in GitHub actions

Pytest is working locally, but failing in GitHub actions:

```bash
============================= test session starts ==============================
platform linux -- Python 3.10.1, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/runner/work/spacypdfreader/spacypdfreader
collected 0 items / 1 error

==================================== ERRORS ====================================
________________ ERROR collecting tests/test_spacypdfreader.py _________________
tests/test_spacypdfreader.py:1: in <module>
    import spacy
../../../.cache/pypoetry/virtualenvs/spacypdfreader-bYjr66NX-py3.10/lib/python3.10/site-packages/spacy/__init__.py:13: in <module>
    from . import pipeline  # noqa: F401
../../../.cache/pypoetry/virtualenvs/spacypdfreader-bYjr66NX-py3.10/lib/python3.10/site-packages/spacy/pipeline/__init__.py:1: in <module>
    from .attributeruler import AttributeRuler
../../../.cache/pypoetry/virtualenvs/spacypdfreader-bYjr66NX-py3.10/lib/python3.10/site-packages/spacy/pipeline/attributeruler.py:6: in <module>
    from .pipe import Pipe
spacy/pipeline/pipe.pyx:1: in init spacy.pipeline.pipe
    ???
E   ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
=========================== short test summary info ============================
ERROR tests/test_spacypdfreader.py - ValueError: numpy.ndarray size changed, ...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.49s ===============================
```

https://github.com/SamEdwardes/spacypdfreader/runs/4754448385?check_suite_focus=true

--- a/.github/workflows/pytest.yml
+++ b/.github/workflows/pytest.yml
@@ -26,6 +26,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
+        sudo apt-get install -y poppler-utils
         curl -sSL https://install.python-poetry.org | python3 -
         poetry export --without-hashes --dev --extras pytesseract --output requirements.txt
         python -m pip install --upgrade pip

--- a/.github/workflows/pytest.yml
+++ b/.github/workflows/pytest.yml
@@ -24,12 +24,17 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install OCR and PDF dependencies
       run: |
         sudo apt-get install -y poppler-utils
+        sudo apt install tesseract-ocr -y
+        sudo apt install libtesseract-dev -y
+    - name: Install python dependencies
+      run: |
         curl -sSL https://install.python-poetry.org | python3 -
         poetry export --without-hashes --dev --extras pytesseract --output requirements.txt
         python -m pip install --upgrade pip
+        pip install wheel
         pip install -r requirements.txt
     - name: Test with pytest
       run: |

--- a/.github/workflows/pytest.yml
+++ b/.github/workflows/pytest.yml
@@ -14,17 +14,22 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.10
+    - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
-        python-version: "3.10"
+        python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
         curl -sSL https://install.python-poetry.org | python3 -
-        poetry install -E pytesseract
+        poetry export --without-hashes --dev --extras pytesseract --output requirements.txt
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
     - name: Test with pytest
       run: |
-        poetry run pytest
+        pytest

